### ISSUE TITLE
Add a yes/no QML element/dialog

### ISSUE THREAD

**Issue by M4rtinK** [M4rtinK/modrana#228]
Mainly to confirm potentially important decisions (delete POI X ? yes/no).

Given the application architecture, it could be a page that asks the question and then pops itself from the stack once an option has been selected.

Such an element could be also useful in Tsubame.

**Comment by Karry** [comment]
Ahoj Martine

Sailfish Silica already has component for this - [`Remorse`](https://sailfishos.org/develop/docs/silica/qml-sailfishsilica-sailfish-silica-remorse.html/). It will shows cancelable countdown before real action. I suggest to use this "native" behaviour. You can find usage on `ManuPage.qml` in ComonentGallery app from SDK...

```
    RemorsePopup {
        id: clearRemorse
    }
    SilicaListView {
        PullDownMenu {
            MenuItem {
                text: "Clear"
                visible: listView.count
                onClicked: clearRemorse.execute("Clearing", function() { listModel.clear() } )
            }
        }
    }
```

**Comment by M4rtinK** [comment]
Yeah, I know about the remorse timer - but frankly, I never liked the concept. It seems just too fragile & needlessly introduces a delay:

- what happens if you swipe back when the timer is running ? should it trigger at once/trigger at the correct time/not trigger ? seems kinda like undefined behavior to me
- you introduce at least a 5 second delay for each decision, which in some cases could be quite a bother
- in other cases (some behavior not expected by the given user) 5 seconds might not be enough - you are basically forcing people to read a piece of text and to decide on it in 5 seconds or *bad things will happen*
- whit a dialog this is not an issue - the user can comprehend the decision as long as needed, or just quickly click though when he is sure what's going on and just wants to confirm the decision
- what if two remorse timers get triggered at once ? Warehouse has that bug and apparently the older one will cover the new one (!), which is counting down in the background, but is not visible as long as the older timer is running...

All in all I think the remorse time was one of the more unfortunate UI decisions on Sailfish OS, together with long-press for additional actions (which is pretty undiscoverable).

Also the remorse timer is a Silica exclusive concept not available anywhere else (Qt Quick Controls 1/2). ModRana is a GUI component toolkit independent application thanks to [Universal Components](https://github.com/M4rtinK/universal-components). So if I wanted to use the remorse timer concept I would have to implement a new element to Universal Components, that provides the remorse timer with the Silica backend and provides something similar with the Controls backend.

I don't think re-implementing the remorse timer so that it shows up to unsuspecting users on desktop and Android would be a good idea, so I would have to use some sort of fallback dialog anyway.

So it just seems to me easier to use a proper dialog that to solve the much more complicated problem of using the (IMHO pretty broken) remorse timer concept in a multi platform way.

**Comment by M4rtinK** [comment]
Implemented in ae92954b36e05cd6e1d27a04f4c3cef8629ec02c.
